java/src/main/java/io/cucumber/gherkin/utils/pretty/PrettyHandlers.java

```diff
@@ -136,6 +136,14 @@ private static String prettyLanguageHeader(String language) {
         return "en".equals(language) ? "" : String.format("# language: %s\n", language);
     }
 
+    private String semiColumnAndName(String name) {
+    	if (name == null || name.isEmpty()) {
```

Comment:
Can name ever be null?

Reply:
I preferred staying on the safe-side, and followed https://github.com/cucumber/gherkin-utils/blob/main/java/src/main/java/io/cucumber/gherkin/utils/pretty/PrettyHandlers.java#L219